<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>playerControlDialog</class>
 <widget class="QDialog" name="playerControlDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Player Integrated Capture</string>
  </property>
  <widget class="QCheckBox" name="lockControlsCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Lock physical controls</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="playerControlsGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>281</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Player controls:</string>
   </property>
   <widget class="QPushButton" name="stopPushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="scanBackwardsPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Scan &lt;</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="stepBackwardsPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Step &lt;</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="scanForwardsPushButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Scan &gt;</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="stepForwardsPushButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Step &gt;</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="playPushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="cavControlsGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>CAV controls:</string>
   </property>
   <widget class="QLineEdit" name="seekPositionLineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="seekPushButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Seek</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="seekLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="playerInformationGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Player information:</string>
   </property>
   <widget class="QLabel" name="staticLabel_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Position:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="staticLabel_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="staticLabel_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Disc type:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="positionInfoLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusInfoLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="discTypeInfoLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="clvControlsGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>CLV controls:</string>
   </property>
   <widget class="QTimeEdit" name="timeSeektimeEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>1999</year>
      <month>12</month>
      <day>20</day>
     </datetime>
    </property>
    <property name="maximumDateTime">
     <datetime>
      <hour>1</hour>
      <minute>59</minute>
      <second>59</second>
      <year>1999</year>
      <month>12</month>
      <day>20</day>
     </datetime>
    </property>
    <property name="minimumTime">
     <time>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
     </time>
    </property>
    <property name="currentSection">
     <enum>QDateTimeEdit::MinuteSection</enum>
    </property>
    <property name="displayFormat">
     <string>HH:mm:ss</string>
    </property>
    <property name="timeSpec">
     <enum>Qt::LocalTime</enum>
    </property>
    <property name="time">
     <time>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
     </time>
    </property>
   </widget>
   <widget class="QPushButton" name="timeSeekPushButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Seek</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeSeeklabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Time:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
